packages/core/src/routes/sign-in-experience.test.ts

```diff
@@ -20,48 +55,91 @@ jest.mock('@/queries/sign-in-experience', () => ({
   ),
 }));
 
-describe('signInExperiences routes', () => {
-  const signInExperienceRequester = createRequester({ authedRoutes: signInExperiencesRoutes });
+const signInExperienceRequester = createRequester({ authedRoutes: signInExperiencesRoutes });
 
-  it('GET /sign-in-exp', async () => {
-    const response = await signInExperienceRequester.get('/sign-in-exp');
-    expect(response.status).toEqual(200);
-    expect(response.body).toEqual(mockSignInExperience);
-  });
+const defaultSignInMethods: SignInMethods = {
+  username: SignInMethodState.primary,
+  email: SignInMethodState.disabled,
+  sms: SignInMethodState.disabled,
+  social: SignInMethodState.disabled,
+};
 
-  it('PATCH /sign-in-exp', async () => {
-    const branding: Branding = {
-      primaryColor: '#000',
-      backgroundColor: '#fff',
-      darkMode: true,
-      darkBackgroundColor: '#000',
-      darkPrimaryColor: '#fff',
-      style: BrandingStyle.Logo,
-      logoUrl: 'http://silverhand.png',
-      slogan: 'silverhand',
-    };
+it('GET /sign-in-exp', async () => {
+  const response = await signInExperienceRequester.get('/sign-in-exp');
+  expect(response.status).toEqual(200);
+  expect(response.body).toEqual(mockSignInExperience);
+});
+
+describe('PATCH /sign-in-exp', () => {
+  describe('should fail without enabled connector when enabled sign-in', () => {
```

Comment:
nit: this sentence is kind of wired.

Reply:
Updated, PTAL

`should fail when enabled sign-in and there is no enabled connector`